dex/pair/src/liquidity_pool.rs

```diff
@@ -10,148 +15,163 @@ const MINIMUM_LIQUIDITY: u64 = 1_000;
 pub trait LiquidityPoolModule:
     amm::AmmModule + config::ConfigModule + token_send::TokenSendModule
 {
-    fn pool_add_liquidity(
-        &self,
-        first_token_amount: BigUint,
-        second_token_amount: BigUint,
-    ) -> SCResult<BigUint> {
-        let first_token = self.first_token_id().get();
-        let second_token = self.second_token_id().get();
-        let total_supply = self.get_total_lp_token_supply();
-        let mut first_token_reserve = self.pair_reserve(&first_token).get();
-        let mut second_token_reserve = self.pair_reserve(&second_token).get();
+    fn pool_add_liquidity(&self, context: &mut AddLiquidityContext<Self::Api>) {
+        let zero = &BigUint::zero();
```

Comment:
can we define zero as constant ?

Reply:
at the moment, no, i suggested something similar to Andrei already, we is working on it